lib/logging.dart

```diff
@@ -88,29 +88,29 @@ class Logger {
   Logger._internal(this.name, this.parent, Map<String, Logger> children)
       : this._children = children,
         this.children = new UnmodifiableMapView(children) {
-    if (parent != null) parent._children[name] = this;
+    /* if (parent != null) */ parent._children[name] = this;
```

Comment:
This "if" test was erroneously commented out because the migration engine incorrectly believed that `Logger.parent` would be non-nullable.  See line 67.